### ISSUE TITLE
required variable index removed as per 'for-index-underscore'

### ISSUE THREAD

**Issue by fametrano** [sourcery-ai/sourcery#72]
Sourcery removes the required variable index `i` from a for loop ('for-index-underscore') in the [tonelli function of btclib.numbertheory](https://github.com/btclib-org/btclib/blob/master/btclib/numbertheory.py#L154).

```
for i in range(1, m):
    t2i = t2i * t2i % p
    if t2i == 1:
        break

# Update next value to iterate
b = pow(c, 1 << (m - i - 1), p)
```

The function has been marked with `# sourcery skip: for-index-underscore` to prevent this.

This bug is present in VSCode extension 0.7.9 and the current GitHub bot.

**Comment by Hellebore** [comment]
Definite bug - thanks for raising!

**Comment by Hellebore** [comment]
fix found - will be in the next release.

**Comment by fametrano** [comment]
> fix found - will be in the next release.

thank you. Should I close the issue now, or you will do it after next release?

**Comment by Hellebore** [comment]
I'll close it after the next release - should be in the next few days.

**Comment by Hellebore** [comment]
Fixed with release of 0.7.11
